posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md

```diff
@@ -0,0 +1,457 @@
+import { MatButtonToggleModule } from '@angular/material/button-toggle';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatCardModule } from '@angular/material/card';
+import { MatDividerModule } from '@angular/material/divider';
+import { MatTableModule } from '@angular/material/table';
+ 
+@NgModule({
+ imports: [
+   MatIconModule,
+   MatButtonModule,
+   MatButtonToggleModule,
+   MatFormFieldModule,
+   MatInputModule,
+   MatCardModule,
+   MatDividerModule,
+   MatTableModule,
+   ReactiveFormsModule,
+   CommonModule,
+   RouterModule.forChild([beersRoute])
+ ],
+ declarations: [BeersComponent],
+})
+export class BeersModule {}
+```
+
+The beers module declares the new component and also adds a new module for a material design table
+
+`beers.component.css`
+
+```css
+mat-card {
+   width: 350px;
+   display: block;
+   margin-left: auto;
+   margin-right: auto;
+   margin-top: 20px;
+ }
+```
+
+A little style copied from the generated code
+
+`beers.component.html`
+
+```html
+<div>
+   <h2>Beer is the answer</h2>
+ </div>
+  <table mat-table [dataSource]="beers" class="mat-elevation-z8" aria-describedby="Beers table">
+   <ng-container matColumnDef="id">
+     <th mat-header-cell *matHeaderCellDef>ID</th>
+     <td mat-cell *matCellDef="let element">{{ element.id }}</td>
+   </ng-container>
+    <ng-container matColumnDef="name">
+     <th mat-header-cell *matHeaderCellDef>Name</th>
+     <td mat-cell *matCellDef="let element">{{ element.name }}</td>
+   </ng-container>
+    <ng-container matColumnDef="unitPrice">
+     <th mat-header-cell *matHeaderCellDef>Unit Price</th>
+     <td mat-cell *matCellDef="let element">{{ element.unitPrice }}</td>
+   </ng-container>
+    <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+   <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
+ </table>
+ ```
+
+ An HTML fragment for displaying the beer data in a table
+
+## Modify these files
+
+The `app-routing.module.ts` file needs to have the beers route added. The entire file should look like this:
+
+```typescript
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+ 
+export const routes: Routes = [
+ {
+   path: '',
+   loadChildren: () => import('./login/login.module').then(m => m.LoginModule),
+ },
+ {
+   path: 'beers',
+   loadChildren: () => import('./beers/beers.module').then(m => m.BeersModule),
+ },
+ 
+// jhipster-needle-angular-route
+];
+ 
+@NgModule({
+ imports: [RouterModule.forRoot(routes)],
+ exports: [RouterModule],
+})
+export class AppRoutingModule {}
+```
+
+The last step is to add a link to the home page at `app.component.html`
+
+```html
+ <a routerLink="beers" routerLinkActive="active" ariaCurrentWhenActive="page">I love beer</a>
+ ```
+
+ You can now restart and add some beers to the service using the steps above
```

Comment:
Same as the other comment. I would be very surprised if JHipster Lite didn't have live reload built in to monitor code changes in the projects. Should be able to "refresh and add some beers"